comment/tests/test_api.py

```diff
@@ -659,12 +659,17 @@ def test_get_profile_model(self):
 
     def test_get_user_fields(self):
         fields = get_user_fields()
-        self.assertEqual(fields, ('id', 'username', 'email', 'profile'))
+        self.assertEqual(fields, ['id', 'username', 'email', 'profile'])
+
+        # change fields
+        patch.object(settings, 'COMMENT_USER_API_FIELDS', ['email', ]).start()
+        fields = get_user_fields()
+        self.assertEqual(fields, ['email', 'profile', ])
 
         mocked_hasattr = patch('comment.api.serializers.hasattr').start()
         mocked_hasattr.return_value = True
         fields = get_user_fields()
-        self.assertEqual(fields, ('id', 'username', 'email', 'profile', 'logentry'))
+        self.assertEqual(fields, ['email', 'profile'])
```

Comment:
Can you separate this to different functions? It would be a bit easier to read and understand what each test wants to accomplish that way.